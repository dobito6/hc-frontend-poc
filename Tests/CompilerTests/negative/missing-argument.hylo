//! no-std

fun f(x: Void = (), y: Void) {}

fun main() {
  f(x: (), y: ())
  f(y: ())
  f()
}
