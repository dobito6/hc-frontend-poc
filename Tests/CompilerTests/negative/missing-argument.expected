negative/missing-argument.hylo:8.3-6: error: incompatible labels: found '()', expected '(x:y:)'
  f()
  ~~~
